types/node/index.d.ts

```diff
@@ -1627,6 +1628,8 @@ declare module "repl" {
         prependOnceListener(event: "reset", listener: (...args: any[]) => void): this;
     }
 
+    export function start(): REPLServer;
+    export function start(prompt: string): REPLServer;
```

Comment:
You could consider to write this as a single line using `options?: string | ReplOptions`.